packages/lesswrong/components/tagging/subforums/RightSidebarColumn.tsx

```diff
@@ -1,92 +1,111 @@
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { MAX_COLUMN_WIDTH } from '../../posts/PostsPage/PostsPage';
 import { TAB_NAVIGATION_MENU_WIDTH } from '../../common/TabNavigationMenu/TabNavigationMenu';
-import { registerComponent } from '../../../lib/vulcan-lib';
+import { Components, registerComponent } from '../../../lib/vulcan-lib';
 
 const MIN_SIDEBAR_WIDTH = 250
 const MAX_SIDEBAR_WIDTH = 370
 const MIN_GAP = 20
 
+const TITLE_HEIGHT_DESKTOP = 245 // ~exactly right
+const TITLE_HEIGHT_MOBILE = 210 // slightly oversized to account for possible wrapping
+
+const Z_TOP = 3
+const Z_IMAGE = 1
+const Z_OVERLAY = 2
+
+const gridTemplateColumns = `
+  minmax(${TAB_NAVIGATION_MENU_WIDTH + MIN_GAP}px, ${TAB_NAVIGATION_MENU_WIDTH + 250}px)
+  0.7fr
+  minmax(${MAX_COLUMN_WIDTH - 20}px, ${MAX_COLUMN_WIDTH}px)
+  minmax(${MIN_GAP}px, 70px)
+  minmax(${MIN_SIDEBAR_WIDTH}px, ${MAX_SIDEBAR_WIDTH}px)
+  minmax(${MIN_GAP}px, 50px)
+  1fr
+`
+
 export const styles = (theme: ThemeType): JssStyles => ({
-  root: {
-    position: "relative",
-    [theme.breakpoints.down('sm')]: {
-      paddingTop: 12
-    },
+  titleWrapper: {
+    overflow: 'hidden',
     [theme.breakpoints.down('md')]: {
       display: 'block !important',
     },
     // Check for support for template areas before applying
     '@supports (grid-template-areas: "title")': {
       display: 'grid',
-      gridTemplateColumns: `
-        minmax(${TAB_NAVIGATION_MENU_WIDTH + MIN_GAP}px, ${TAB_NAVIGATION_MENU_WIDTH + 250}px)
-        0.7fr
-        minmax(${MAX_COLUMN_WIDTH - 20}px, ${MAX_COLUMN_WIDTH}px)
-        minmax(${MIN_GAP}px, 70px)
-        minmax(${MIN_SIDEBAR_WIDTH}px, ${MAX_SIDEBAR_WIDTH}px)
-        minmax(${MIN_GAP}px, 50px)
-        1fr
-      `,
+      gridTemplateColumns: gridTemplateColumns,
       gridTemplateAreas: `
         "... ... title   .... ....... .... ..."
-        "... ... content gap1 sidebar gap2 ..."
       `,
     },
   },
+  bannerImageWrapper: {
+    position: 'absolute',
+    width: '100%',
+    zIndex: Z_IMAGE,
+  },
+  bannerImage: {
+    width: '100%',
+    objectFit: 'cover',
+    height: TITLE_HEIGHT_DESKTOP,
+    [theme.breakpoints.down('sm')]: {
+      height: TITLE_HEIGHT_MOBILE,
+    }
+  },
+  translucentOverlay: {
+    position: 'absolute',
+    zIndex: Z_OVERLAY,
+    width: '100%',
+    height: TITLE_HEIGHT_DESKTOP,
+    backgroundColor: 'rgba(0,0,0,0.4)', // hardcode overlay to keep it the same in light and dark mode
```

Comment:
I think the right thing to do is add this new color to the palette and set it to be the same in light and dark mode